entrypoint.sh

```diff
@@ -33,13 +33,20 @@ if [[ -v INPUT_INPUT_TYPE && -n "${INPUT_INPUT_TYPE}" ]]; then
   REGULA_OPTS+=("-t" "${INPUT_INPUT_TYPE}")
 fi
 
+if [[ -v INPUT_OUTPUT_FORMAT && -n "${INPUT_OUTPUT_FORMAT}" ]]; then
+  REGULA_OPTS+=("-f" "${INPUT_OUTPUT_FORMAT}")
+fi
+
 EXIT_CODE=0
-REGULA_OUTPUT=$(cd "$GITHUB_WORKSPACE" && regula run -f json ${REGULA_OPTS[@]} $INPUT_PATH) ||
+REGULA_OUTPUT=$(cd "$GITHUB_WORKSPACE" && regula run ${REGULA_OPTS[@]} $INPUT_PATH) ||
```

Comment:
Sorry for the late response, and good to see you Eric!  I was wondering if we could change this to run Regula twice if the output format is not JSON.  That way, we can consistently set the `RULES_PASSED/FAILED` variables.  It could be a bit confusing to a user if an output they are using disappears when they change the output format.